src/Http/Controllers/Admin/MediaLibraryController.php

```diff
@@ -289,4 +303,21 @@ private function getExtraMetadatas()
             return [$field['name'] => $fieldInRequest];
         });
     }
+
+    /**
+     * @param $sasUrl
+     * @return JsonResponse
+     */
+    public function isValidSas($sasUrl)
+    {
+        return response($sasUrl);
```

Comment:
The injected `responseFactory` should be preferred over the `response` helper, like in other methods returning responses in this class.